ads/google/a4a/utils.js

```diff
@@ -65,19 +65,18 @@ export function isGoogleAdsA4AValidEnvironment(win, element) {
  * @param {!../../../extensions/amp-a4a/0.1/amp-a4a.AmpA4A} a4a
  * @param {string} baseUrl
  * @param {number} startTime
- * @param {number} slotNumber
  * @param {!Array<!./url-builder.QueryParameterDef>} queryParams
  * @param {!Array<!./url-builder.QueryParameterDef>} unboundedQueryParams
  * @return {!Promise<string>}
  */
 export function googleAdUrl(
-    a4a, baseUrl, startTime, slotNumber, queryParams, unboundedQueryParams) {
+    a4a, baseUrl, startTime, queryParams, unboundedQueryParams) {
   /** @const {!Promise<string>} */
   const referrerPromise = viewerForDoc(a4a.getAmpDoc()).getReferrerUrl();
```

Comment:
Would it be possible/possible to recover from either of these promises failing?  Does it occur in the wild?  I'm trying to be much more explicit about possible error conditions and how we would handle/recover